src/components/Footer.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+
+// MUI UI/UX styles
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import Grid from "@mui/material/Grid";
+import Link from "@mui/material/Link";
+import ImageListItem from "@mui/material/ImageListItem";
+
+const Footer = () => {
+    return (
+        <footer>
+            <Box sx={styles.footerBox}>
+                <Container sx={styles.container}>
+                    <Grid container spacing={4}>
+                        <Grid item xs={12} sm={3}>
+                            <ImageListItem sx={styles.logo}>
+                                <img
+                                    src="https://images.pexels.com/photos/1279813/pexels-photo-1279813.jpeg?auto=compress&cs=tinysrgb&w=300&h=300&dpr=1"
+                                    alt="devprep-logo"
+                                />
+                            </ImageListItem>
+                        </Grid>
+                        <Grid item xs={12} sm={3} lineHeight={2}>
+                            <Box sx={styles.bottomLine}>ABOUT US</Box>
+                            <Box>
```

Comment:
The links in the foot can use <Button component={link} to="/about-us">About Us</Button>.  For the links that have not been defined in the router yet, just use to="/", and use '// TODO - the url will be update later.'

Reply:
Thanks for the detailed feedback. This section has been modified